examples/avatar/agent_worker.py

```diff
@@ -0,0 +1,110 @@
+import argparse
+import logging
+import sys
+from dataclasses import asdict, dataclass
+from functools import partial
+
+import httpx
+from dotenv import load_dotenv
+from livekit import api
+from livekit.agents import JobContext, WorkerOptions, WorkerType, cli
+from livekit.agents.pipeline import AgentTask, PipelineAgent
+from livekit.agents.pipeline.datastream_io import DataStreamOutput
+from livekit.agents.pipeline.io import PlaybackFinishedEvent
+from livekit.agents.pipeline.room_io import RoomInput, RoomInputOptions
+from livekit.plugins import openai
+
+logger = logging.getLogger("avatar-example")
+logger.setLevel(logging.INFO)
+
+load_dotenv()
+
+
+AVATAR_IDENTITY = "avatar_worker"
+
+
+@dataclass
+class AvatarConnectionInfo:
+    room_name: str
+    url: str  # LiveKit server URL
+    token: str  # Token for avatar worker to join
+
+
+async def launch_avatar_worker(
+    ctx: JobContext, avatar_dispatcher_url: str, avatar_identity: str
+) -> None:
+    """Wait for worker participant to join and start streaming"""
+    # create a token for the avatar worker
+    # TODO(long): do we need to set agent=True here? in playground if not the video track is not automatically displayed
+    token = (
+        api.AccessToken()
+        .with_identity(avatar_identity)
+        .with_name("Avatar Worker")
+        .with_grants(api.VideoGrants(room_join=True, room=ctx.room.name, agent=True))
+        .with_metadata("avatar_worker")
+        .to_jwt()
```

Comment:
we should specify the identity of the "original" agent as well so that the integrator can add a guard on their RPC handlers methods (`if data.caller_identity !== agent_identity:  print("RPC call from unexpected participant"); return`)